specs/protocol/deposits.md

```diff
@@ -287,6 +286,50 @@ and also set the following new attributes:
   Or `1` if the L1 block does not support blobs.
   The `1` value is derived from the EIP-4844 `MIN_BLOB_GASPRICE`.
 
+#### L1 Attributes - Fjord
+
+On the Fjord activation block, and if Fjord is not activated at Genesis,
+the L1 Attributes Transaction includes a call to `setL1BlockValuesEcotone()`
+because the L1 Attributes transaction precedes the [Fjord Upgrade Transactions][fjord-upgrade-txs],
+meaning that `setL1BlockValuesFjord` is not guaranteed to exist yet.
+
+Every subsequent L1 Attributes transaction should include a call to the `setL1BlockValuesFjord()` function.
+The input args are no longer ABI encoded function parameters,
+but are instead packed into 5 32-byte aligned segments and one 12-byte segment (starting after the function selector).
+Each signed and unsigned integer argument is encoded as big-endian using a number of bytes corresponding to the
+underlying type. The overall calldata layout is as follows:
+
+[fjord-upgrade-txs]: derivation.md#fjord
+
+| Input arg         | Type    | Calldata bytes | Segment |
+|-------------------|---------|----------------|---------|
+| {0x850c16d8}      |         | 0-3            | n/a     |
+| baseFeeScalar     | uint32  | 4-7            | 1       |
+| blobBaseFeeScalar | uint32  | 8-11           |         |
+| sequenceNumber    | uint64  | 12-19          |         |
+| l1BlockTimestamp  | uint64  | 20-27          |         |
+| l1BlockNumber     | uint64  | 28-35          |         |
+| basefee           | uint256 | 36-67          | 2       |
+| blobBaseFee       | uint256 | 68-99          | 3       |
+| l1BlockHash       | bytes32 | 100-131        | 4       |
+| batcherHash       | bytes32 | 132-163        | 5       |
+| l1CostIntercept   | int32   | 164-167        | 6       |
+| l1CostFastlzCoef  | int32   | 168-171        |         |
+| l1CostTxSizeCoef  | int32   | 172-175        |         |
+
+Total calldata length MUST be exactly 176 bytes. This helps to slow database growth as every
```

Comment:
The second sentence here doesn't feel useful. `This helps to slow database growth as...` not super clear what the alternative is? It also *adds* history size based on ecotone

Reply:
agreed (this is a copy/paste from above), i will remove it